### PR TITLE
fix: no packages found case

### PR DESCRIPTION
**Pull request by lime-green**

### CODE REVIEW

File: src/command-helpers.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const lernaExecutable = path.join(
 
 function getLernaUpdatedJson() {
     return exec(`${lernaExecutable} updated --json`).catch(err => {
-        if (err.message.includes('No packages need updating')) {
+        if (err.message.includes('No changed packages found')) {
```
Comment (msrose) [inline src/command-helpers.js]: Could possibly do an `||` here to support both messages?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "monodeploy",
-    "version": "0.0.9",
+    "version": "0.0.10",
```
Comment (msrose) [inline package.json]: Let's leave this out for now, once the PR gets merged we can do a release from the master branch by doing `yarn version --patch` and then `git push origin --tags master`

File: src/command-helpers.js (hunk 1 above)
Comment (lime-green) [reply src/command-helpers.js]: why? This was the message on the old version which we're no longer using

Comment (msrose) [reply src/command-helpers.js]: My instinct just says to keep trying to support the fe-libraries version until we can get it upgraded there, but yeah I guess it's not super critical.

**Pull request merged**